examples/08-DeepAR-examples.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p>can you rerun this notebook? seems like there were some issues with too short series for validation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/2008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>